resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -165,7 +166,9 @@ private[spark] object Config extends Logging {
         "OUTLIER policy chooses an executor with outstanding statistics which is bigger than" +
         "at least two standard deviation from the mean in average task time, " +
         "total task time, total task GC time, and the number of failed tasks if exists. " +
-        "If there is no outlier, it works like TOTAL_DURATION policy.")
+        "If there is no outlier then no executor will be rolled." +
+        "OUTLIER_OR_TOTAL_DURATION policy picks an outlier using the OUTLIER policy above. " +
```

Comment:
Could you keep the previous behavior by making `OUTLIER_OR_TOTAL_DURATION` as the default?
```scala
- .createWithDefault(ExecutorRollPolicy.OUTLIER.toString)
+ .createWithDefault(ExecutorRollPolicy.OUTLIER_OR_TOTAL_DURATION.toString) 
```